cou/apps/auxiliary.py

```diff
@@ -63,13 +63,13 @@ def is_valid_track(self, charm_channel: str) -> bool:
             current_track,
         ) in TRACK_TO_OPENSTACK_MAPPING and len(possible_tracks) > 0
 
-    @property
-    def expected_current_channel(self) -> str:
+    def expected_current_channel(self, _target: OpenStackRelease) -> str:
```

Comment:
not-blocking: What is reason to use `_target` instead of `target`? Why it's not part of docstring?

Reply:
This actually good trick I did not know about :+1: